docs/content/how-to/aws/disaster-recovery.md

```diff
@@ -1,17 +1,48 @@
 ---
-title: Migrating Hosted Cluster among the same AWS Region
+title: Disaster Recovery
 ---
 
-# Migrating Hosted Cluster among the same AWS Region
+# Disaster Recovery
 
-## Preface and Considerations
+## Migrating Hosted Cluster among the same AWS Region
+
+### Use cases for Disaster Recovery
+
+We want to help people to understand what situations could be considered a fit for this disaster recovery procedure.
+
+1. *The control-plane is down for your hosted cluster (api-server, etcd,...)*
+- For the first situation this procedure it's a perfect fit.
+
+2. *The management cluster API-server/etcd is down?*
+- For this second situation it does not make sense to use this procedure, so you can use a backup/recovery tool like Velero in order to recover the ETCD.
+
+3. *Hypershift operator is not working, it’s down and it cannot be recovered*
+- For this situation the procedure it's a perfect fit.
+
+4. *MGMT Cluster capacity its overbooked and the management cluster it’s failing because of lack of vCpus/mem or something like that.*
```

Comment:
This doesn't seem like a good use case to me. In this scenario you should just scale out your management cluster data plane. Can we drop it for now?

Reply:
Let me rewrite instead of drop that out, because I would like to use it in the nexts examples

Reply:
I still don't understand "MGMT Cluster capacity its overbooked". The hard limit we have today it's mostly in on aws quotas. If a management cluster needs more capacity you just create more workers to allocated more HostedClusters. Unless you mean the management cluster control plane capacity in which case this falls under 2. So unless there's a clear definition of "MGMT Cluster capacity its overbooked" that can't be solved creating more workers and that is different 2 I still suggest to drop it.

Reply:
I've removed this fourth bullet point, also the mixed scenarios part.